PineapplePlanner/PineapplePlanner.UI/Pages/Home.razor

```diff
@@ -129,3 +130,11 @@
         }
     </div>
 </div>
+
+@code {
+
+    private string GetFilterString(List<string> selectedValues)
+    {
+        return string.Join(", ", selectedValues.Select(x => _localize[(String)x]));
```

Comment:
its preferred to use `string` instead of `String` or simply call `toString()`. but in that case none of that is needed because x is a `string` anway.